libsql-ffi/bundled/src/sqlite3.c

```diff
@@ -69225,7 +69330,7 @@ static int sqlite3WalOpen(
     out->methods.xSavepoint = (void (*)(wal_impl *, unsigned int *))sqlite3WalSavepoint;
     out->methods.xSavepointUndo = (int (*)(wal_impl *, unsigned int *))sqlite3WalSavepointUndo;
     out->methods.xFrames = (int (*)(wal_impl *, int, libsql_pghdr *, unsigned int, int, int))sqlite3WalFrames;
-    out->methods.xCheckpoint = (int (*)(wal_impl *, sqlite3 *, int, int (*)(void *), void *, int, int, unsigned char *, int *, int *))sqlite3WalCheckpoint;
+    out->methods.xCheckpoint = (int (*)(wal_impl *, sqlite3 *, int, int (*)(void *), void *, int, int, unsigned char *, int *, int *, void*, int (*)(void*, int, const unsigned char*, int, int, int)))sqlite3WalCheckpoint;
```

Comment:
note: this needs to be documented somewhere, we're changing user-facing public API